control-plane/subcommand/connect-init/command.go

```diff
@@ -182,6 +169,15 @@ func (c *Command) Run(args []string) int {
 			return 1
 		}
 	}
+
+	// todo (agentless): this should eventually be passed to consul-dataplane as a string so we don't need to write it to file.
+	if c.consul.UseTLS && c.consul.CACertPEM != "" {
+		if err = common.WriteFileWithPerms("/consul/connect-inject/consul-ca.pem", c.consul.CACertPEM, 0444); err != nil {
```

Comment:
time for my fave nit: can we extract the ca file path to a constant?

Reply:
done! It's a great nit because that allowed us to unify the file we're passing to consul-dataplane command